src/Twilio/Http/SystemNetHttpClient.cs

```diff
@@ -36,62 +32,41 @@ public SystemNetHttpClient(System.Net.Http.HttpClient httpClient = null)
         /// <returns>Twilio response</returns>
         public override Response MakeRequest(Request request)
         {
-            var task = MakeRequestAysnc(request);
-            task.Wait();
-            return task.Result;
+            try
+            {
+                var task = MakeRequestAsync(request);
+                task.Wait();
+                return task.Result;
+            }
+            catch (AggregateException ae)
+            {
+                // Combine nested AggregateExceptions
+                ae = ae.Flatten();
+                throw (ae.InnerExceptions.Count == 1) ? ae.InnerException : ae;
```

Comment:
⚠️  *Question*: Should we always return the first exception? Cant think of a scenario when wed have multiple exceptions in the AggregateException but its technically possible. Throwing an AggregateException leaks the implementation detail that the synchronous MakeRequest calls the async version under the covers.